code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -559,7 +579,7 @@
 	icon_state = "pineapple_slice"
 	filling_color = "#F6CB0B"
 	tastes = list("pineapple" = 1)
-	foodtype = FRUIT | PINEAPPLE
+	foodtype = FRUIT
```

Comment:
🤔

Reply:
It may of been from a earlier version of the source because I noticed there were some changes and I just copied my source code, where I should of just cut and pasted it in places instead of the whole file. I forked the latest version just moments ago so it probably had changes from the version I downloaded earlier today.